packages/kolibri-common/strings/enhancedQuizManagementStrings.js

```diff
@@ -216,14 +209,14 @@ export const enhancedQuizManagementStrings = createTranslator('EnhancedQuizManag
   },
   cannotSelectSomeTopicWarning: {
     message:
-      'You can only select folders with { count, number } or less exercises and no subfolders to avoid oversized quizzes.',
+      'You can only select folders with 12 or fewer exercises.',
```

Comment:
@rtibbles @nucleogenesis - do we want to keep this (and the "max number of questions 50" string) with this flexible syntax? maybe it's worth it it just seemed like we are not actually planning to add dynamic counts in either place

Reply:
Yes, I deliberately updated it from the static string that was here previously to give us the flexibility to change this to whatever made the most sense.

Reply:
The other thing here is whether this criterion is even the correct one - we could change it to just do it based on the number of questions available in the folder.